libs/portal/shared/features/form/ui/combobox-field.tsx

```
@@ -137,7 +141,9 @@ export function ComboboxField({
                                 field.value === option.value ? 'opacity-100' : 'opacity-0',
                               )}
                             />
-                            {option.label}
+                            <button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)